src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write code here
+const listItems = [...document.querySelectorAll('li')];
+const employeesList = document.querySelector('ul');
+
+function convertToNumber(string) {
+  const salaryNumber = string.replace('$', '').replace(',', '');
+
+  return +salaryNumber;
+}
+
+function sortBySalary(items) {
+  items.sort(
+    (a, b) =>
+      convertToNumber(b.dataset.salary) - convertToNumber(a.dataset.salary),
+  );
```

Comment:
It's a good practice to return the result of the sort function. While JavaScript's Array.sort() method does sort the array in-place, it's considered good practice to return the sorted array from the function for clarity and to allow for method chaining.